lms/djangoapps/edxnotes/helpers.py

```diff
@@ -141,15 +137,13 @@ def preprocess_collection(user, course, collection):
     store = modulestore()
     filtered_collection = list()
     cache = {}
+    include_extra_info = settings.NOTES_DISABLED_TABS == []
```

Comment:
This should be `include_path_info = 'course_structure' in settings.NOTES_DISABLED_TABS`.